modules/nf-core/bcftools/pluginsplit/main.nf

```diff
@@ -8,7 +8,7 @@ process BCFTOOLS_PLUGINSPLIT {
         'biocontainers/bcftools:1.20--h8b25389_0' }"
 
     input:
-    tuple val(meta), path(vcf), path(tbi)
+    tuple val(meta), path(vcf, stageAs: "?/*"), path(tbi, stageAs: "?/*")
```

Comment:
Just to check, do these stage in the same directory?
I would have specified a folder to put them into (like `stageAs: "input/*"`), rather than using `?`.
Can't immediately find the table in the documents that explains this.

Reply:
It seems to stage everything in a folder name `1`.
I did as it was made in the other modules, but I would also prefer to specify the folder.